### PR TITLE
Use ASCII apostrophe instead of the curly one

### PR DESCRIPTION
**Pull request by Youssef1313**
Applied **only** to the architecture folder.

### CODE REVIEW

File: docs/architecture/cloud-native/azure-data-storage.md
Comment (Youssef1313) [inline docs/architecture/cloud-native/azure-data-storage.md]: I feel like the backticks is more suitable here. Can someone confirm ?
I've done the same in other few places.

File: docs/architecture/cloud-native/definition.md
Comment (Youssef1313) [inline docs/architecture/cloud-native/definition.md]: Six months from when ?
I don't think this whole sentence has a good wording.

File: docs/architecture/cloud-native/azure-data-storage.md
Comment (pkulikov) [inline docs/architecture/cloud-native/azure-data-storage.md]: According to the current guidelines, commas and periods should be inside quotation marks:
https://docs.microsoft.com/en-us/style-guide/punctuation/quotation-marks

Comment (Youssef1313) [reply docs/architecture/cloud-native/azure-data-storage.md]: @pkulikov, Does this contradict with #16513?

Comment (pkulikov) [reply docs/architecture/cloud-native/azure-data-storage.md]: @Youssef1313 yes, changes made in #16513 look to contradict the guidelines.

As for #16513 though, I wouldn't use quotation marks there at all. I guess those quoted words would be localized, but they shouldn't as they are parts of members' names which are always in English. I would use backticks there.

Comment (Youssef1313) [reply docs/architecture/cloud-native/azure-data-storage.md]: I agree that they shouldn't be localized, either using backticks or the no-loc metadata.
Regarding the style-guide, I'll wait for @mairaw or @gewarren.
What I understood from it is to usually put the punctuation outside it with the noted exception:

> **Exception** If punctuation is part of the quoted material, place it inside the quotation marks.

Comment (pkulikov) [reply docs/architecture/cloud-native/azure-data-storage.md]: The referenced page mentions commas and periods separately; also, the examples there put both comma and period inside the quotation marks. 

As for the exception, I understand it refers to the question and exclamation marks, for example.

Comment (gewarren) [reply docs/architecture/cloud-native/azure-data-storage.md]: Yes, according to the Microsoft style guide and the Chicago Manual of Style, periods and commas should appear inside the closing quotation mark. This is actually news to me, and I don't like this style :). The Oxford or British style is to always put punctuation outside the closing quotation mark, except if it's part of the quoted material.

Comment (mairaw) [reply docs/architecture/cloud-native/azure-data-storage.md]: I've seen that before but I also dislike that style. 😄

**Comment by mairaw** [comment]
@nishanil please review and address comments and questions. I'm fine with the changes.

File: docs/architecture/cloud-native/azure-data-storage.md
Comment (Youssef1313) [reply docs/architecture/cloud-native/azure-data-storage.md]: @mairaw, Yes. The style looks really strange to me. But if it's required I'll do that.

**Comment by mairaw** [comment]
@Youssef1313 can you resolve the new merge conflict? 
I've asked @nishanil offline to check the comments.

**Comment by Youssef1313** [comment]
@mairaw, I've fixed the merge conflicts. However, I still didn't change the punctuation back inside the double quotes. Do you want me to do that?

**Comment by mairaw** [comment]
Thanks @Youssef1313. I'm checking on that one.

File: docs/architecture/cloud-native/definition.md
Comment (nishanil) [reply docs/architecture/cloud-native/definition.md]: @robvet Can you address this?

File: docs/architecture/cloud-native/azure-data-storage.md
Comment (nishanil) [reply docs/architecture/cloud-native/azure-data-storage.md]: As part of this PR https://github.com/dotnet/docs/pull/15353, this content is moved to a new page `relational-vs-nosql-data`. @robvet I think, backticks is correct. Can you consider this change in the new PR?

File: docs/architecture/cloud-native/definition.md
Comment (mairaw) [reply docs/architecture/cloud-native/definition.md]: A quick fix could be to say “six months later”

Comment (nishanil) [reply docs/architecture/cloud-native/definition.md]: I think, we should drop the entire sentence "Interestingly, six months..". The definition does not change, it's the perception that change and their understanding gets better. I was hoping Rob will come up with an interesting sentence to conclude the para.

**Comment by mairaw** [comment]
Maybe we should drop the changes to the cloud native book from this PR, to not conflict with the PR we're trying to merge this week for #15353. And then after that PR is merged, we can apply the fixes.

**Comment by Youssef1313** [comment]
@mairaw, Is there some git command(s) that allows me to revert those changes in cloud-native only ? Or doing that manually is the only way ?

**Comment by mairaw** [comment]
I think in this case it might have to be manually. Otherwise, we can wait until the other PR is merged and then you can rebase yours.

**Comment by Youssef1313** [comment]
Ok, I'll wait on this

**Comment by gewarren** [comment]
You can do `git checkout upstream/master docs/architecture/cloud-native/.`.

**Comment by robvet** [comment]
Agree with @mairaw , would be good to complete this PR (15353) as it focuses on CH 5.
Perhaps we could address these newer concerns in another PR as they extend beyond CH 5.

**Comment by tdykstra** [comment]
@mairaw regarding punctuation inside quotes

>I'm checking on that one.

If you're still waiting on an answer, my suggestion is to go ahead and move punctuation inside quotes per the style guide guidelines, fix the conflicts, and merge this PR.

**Comment by mairaw** [comment]
There are a lot of merge conflicts here now because of book updates that were since released.

**Comment by robvet** [comment]
Tring to make sense out of 16746.

Not where we are with it.

Rob Vettor  | Principal Cloud-Native Application Architect
                      | ASD IP Architect
                      | CNCF (Cloud-Native Computing Foundation) Speaker Bureau<https://www.cncf.io/speaker/robvet/>
                      | Developer Advisory Services for Azure

[cid:image001.jpg@01D5F0C5.B418B7C0]

[Description: cid:image001.gif@01CC9CCD.D94B9620]  214-707-0584
[Description: cid:image003.gif@01CC9CCD.D94B9620]   robvet@microsoft.com<mailto:robvet@microsoft.com>
[cid:image004.jpg@01D5F0C5.B418B7C0]  www.thinkingincloudnative.com<https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.thinkingincloudnative.com%2F&data=02%7C01%7Crobvet%40microsoft.com%7Ca0ad6989676c41a1c02208d7659ff0d4%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637089609919099893&sdata=Ih3VA8PbdEeYpmeYsH5jWzaoZX6Ys5Rwtpw3UlpdIVA%3D&reserved=0>

Co-Author of…
[cid:image005.png@01D5F0C5.B418B7C0]<https://docs.microsoft.com/en-us/dotnet/architecture/cloud-native/>

**Comment by mairaw** [comment]
Some of the content here @robvet is using the curly brace aposthrophe (’) instead of the ASCII version ('). The recommendation is to use the ASCII one and we did already removed them in the past but some were reintroduced with the new content and updates.

**Comment by BillWagner** [comment]
This has become a bit of a challenge, with merge conflicts and other changes.

@nishanil @mairaw @Youssef1313 What's the best recommendation to resolve?

**Review by gewarren — APPROVED** [review]
Going to merge this now!

**Pull request merged**
